jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -494,15 +513,18 @@ ecma_new_ecma_string_from_magic_string_ex_id (lit_magic_string_ex_id_t id) /**<
 } /* ecma_new_ecma_string_from_magic_string_ex_id */
 
 /**
- * Allocate new ecma-string and fill it with reference to length magic string
+ * Returns the constant assigned to the length magic string.
+ *
+ * Note:
+ *   Calling ecma_deref_ecma_string on the returned pointer is optional.
  *
  * @return pointer to ecma-string descriptor
  */
 ecma_string_t *
-ecma_new_ecma_length_string (void)
+ecma_get_length_string (void)
```

Comment:
Would it make sense to get rid of this function? IMHO there is no real benefit to have this in this form. We could simply call the get_magic_string directly where needed.

Reply:
I will do it in another PR if that is good.

Reply:
#2231